src/backend/BSConfig.pm.template

```diff
@@ -284,6 +284,8 @@ our $publish_containers = [
    '.*'             => ['local'],
 ];
 
+our $local_registry_signatures_extension = 1;
```

Comment:
A comment here would be nice ... esp. the for hint for the apache config update for updaters...